src/main/java/com/memeit/post/Post.java

```diff
@@ -39,12 +37,20 @@ public class Post {
     @Column(name = "UPLOAD_DATE", nullable = false)
     private LocalDate uploadDate;
 
-    @Column(name = "VOTES")
-    private int votes;
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
good job what's going on here?

Reply:
Set is needed to get distinct votes per user. we need to make it as Many to Many relation. Many users can vote for many posts.